packages/test-util/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@lodestar/test-util",
```

Comment:
This directory and package should be called `test-utils`, `@lodestar/test-utils` to match `utils` and `@lodestar/utils`